receiver/awsxrayreceiver/internal/socketconn/socketconn.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
@lizthegrey @ccaraman @sarahnovotny do you know if this is the right way to do this?

Reply:
I would love to know as well. The only reason I put this kind of header in some files is because some of the files are largely a port of the code in the existing X-Ray daemon with some relatively big changes. The X-Ray daemon itself is licensed under Apache v2.

Reply:
@bogdandrutu @lizthegrey @ccaraman @sarahnovotny

I've reached out to @alolita on AWS side regarding the use of the Amazon copyright header. And she will follow up with OT committee.